heron/tools/admin/src/python/main.py

```diff
@@ -0,0 +1,178 @@
+# Copyright 2016 Twitter. All rights reserved.
```

Comment:
s/2016/2017.

This is a new tool? Is there a doc about it?

Reply:
There isn't currently.  Will right one in the future, but the you can always type "heron-admin help" and it will give you a detailed explanation of all commands and subcommads